### PR TITLE
Availability and User mapping

### PR DESCRIPTION
**Pull request by claireche89**
### Description
What new features or functions does this PR provide? What bugs does it fix?
created availability entity, and created one to many relationship between user and availability tables in the database

### Testing
How did you test the code in the pull request?
need to test populating data in availability table

### Checklist
- [ ] I provided a description of the contents of this pull request.
- [ ] I've linked this pull request to an issue.
- [ ] I've assigned myself as the Assignee and labelled the PR correctly.

### CODE REVIEW

File: src/main/java/com/mcitmocks/mcitmocks/User/UserRepository.java [hunk 1]
```diff
@@ -3,10 +3,12 @@
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.Optional;
+import java.util.UUID;
 
-public interface UserRepository extends CrudRepository<User, String> {
+public interface UserRepository extends CrudRepository<User, UUID> {
```
Comment (wilsonplau) [inline src/main/java/com/mcitmocks/mcitmocks/User/UserRepository.java]: If it hasn't been updated in `User`, we shouldn't update it here, probably? I don't mind which one we use, but we should be consistent.

File: src/main/java/com/mcitmocks/mcitmocks/MCITMocksApplication.java [hunk 2]
```diff
@@ -18,5 +22,11 @@ String test () {
     public static void main(String[] args) {
         SpringApplication.run(MCITMocksApplication.class, args);
     }
-
+    @Bean
+    CommandLineRunner commandLineRunner(UserRepository userRepository){
+        return args -> {
+            User mario= new User("mario@seas.upenn.edu","cst");
```
Comment (wilsonplau) [inline src/main/java/com/mcitmocks/mcitmocks/MCITMocksApplication.java]: Just one note - TimeZone will be in standard format like `America/Chicago`

**Review by wilsonplau — COMMENTED** [review]
In general, LGTM. One thing to think about this week, if you have some time, is how we might build a Service class that helps us set/edit these availabilities, for example, given a user and time as an ISO DateTime string. 

The other thing is - this is completely my fault – could you install and set it up to format on save? It will help us all be consistent on formatting. https://plugins.jetbrains.com/plugin/7642-save-actions. Could you update this PR again after you've run the auto-format on all of the changed files?

File: src/main/java/com/mcitmocks/mcitmocks/User/UserRepository.java (hunk 1 above)
Comment (claireche89) [reply src/main/java/com/mcitmocks/mcitmocks/User/UserRepository.java]: you are right. I will update from UUID to String

File: src/main/java/com/mcitmocks/mcitmocks/MCITMocksApplication.java (hunk 2 above)
Comment (claireche89) [reply src/main/java/com/mcitmocks/mcitmocks/MCITMocksApplication.java]: got it! Thanks

**Pull request closed without merging**